Error: Cannot find module 'lodash/array/without'

Getting this error

```
Error: Cannot find module 'lodash/array/without'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (C:\…\node_modules\babel-plugin-class-autobind\lib\index.js:104:16)
    at Module._compile (module.js:570:32)
    at Module._extensions..js (module.js:579:10)
    at Object.require.extensions.(anonymous function) [as .js] (C:\…\node_modules\babel-register\lib\node.js:152:7)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at C:\…\node_modules\babel-core\lib\transformation\file\options\option-manager.js:174:20
    at Array.map (native)
    at Function.normalisePlugins (C:\…\node_modules\babel-core\lib\transformation\file\options\option-manager.js:154:20)
    at OptionManager.mergeOptions (C:\…\node_modules\babel-core\lib\transformation\file\options\option-manager.js:228:36)
    at OptionManager.init (C:\…\node_modules\babel-core\lib\transformation\file\options\option-manager.js:373:12)
    at compile (C:\…\node_modules\babel-register\lib\node.js:103:45)
    at loader (C:\…\node_modules\babel-register\lib\node.js:144:14)
    at Object.require.extensions.(anonymous function) [as .js] (C:\…\node_modules\babel-register\lib\node.js:154:7)
    at Module.load (module.js:487:32)
```

```
+-- babel-cli@6.16.0
+-- babel-core@6.17.0
+-- babel-register@6.16.3
+-- babel-plugin-class-autobind@1.0.0
```

--- a/package.json
+++ b/package.json
@@ -31,7 +31,8 @@
     "babel-plugin"
   ],
   "dependencies": {
-    "babel-preset-react": "^6.3.13"
+    "babel-preset-react": "^6.3.13",
+    "lodash": "^3.10.1"
   },
   "name": "babel-plugin-class-autobind",
   "directories": {

